### PR TITLE
Another issue #8 attempt

### PR DESCRIPTION
**Pull request by bealecs**
Issue #8 : Jerome said that there were some things that needed changing with the code. Re opened branch 8 and made some changes, new PR created as a result. Awaiting comment from Jerome/another mentor on what fixes the code needs before merge and close

### CODE REVIEW

File: Portfolio/issues/eventListeners.js [hunk 1]
```diff
@@ -5,12 +5,17 @@ const contactButton = document.getElementById("contactID");
 const resumeButton = document.getElementById("resumeID");
 const logoButton = document.getElementById("logotitle");
 
+//buttons for project section
+const pwpProject = document.getElementById("pwpSection");
+const clifCatalog = document.getElementById("clifCatalogSection");
+const ccLighting = document.getElementById("ccLightingSection");
```
Comment (axecopfire) [inline Portfolio/issues/eventListeners.js:11]: nit: 
(nit means you don't have to change anything. Just for your information)

I like to use `camelCase` for Javascript.
I like to use `kebab-case` for html/css.
I woulda made these HTML element Ids like `pwp-section`, `clif-catalog-section`.

File: Portfolio/issues/index.html [hunk 2]
```diff
@@ -24,8 +25,12 @@ <h1 id="logotitle">Clif Codes</h1>
             </ul>
         </nav>
     </header>
+
     <section class="landing-section" id="landingSection">
         <div>
+            <img class="pfp" src="./images/pfp.webp" alt="Professional shot of Clif Beale with a transparent background" width="350px"/>
```
Comment (axecopfire) [inline Portfolio/issues/index.html:31]: Nit:
As your css grows you're going to end up hunting down where things are styled. Having this width defined as an attribute will overwrite any css you write to width. Meaning if you want to change the width on mobile, you won't be able to and will have to hunt this little guy down.

File: Portfolio/issues/index.html [hunk 3]
```diff
@@ -100,7 +104,100 @@ <h3 class="contact-title">Contact Me</h3>
             <button type="submit">Submit</button>
         </form>
     </section>
+
+    <section class="resume-container" id="resumeSection">
+        <h2 class="about-title">My <span class="name2">Resume</span></h2>
+        <h4 class="download-txt">For the full resume, click below to download</h4>
+        <a class="nounderline" href="./pdf/Clifton Beale's Resume (Dice).pdf" target="_blank" rel="noopener noreferrer" download><button>Download</button></a>
```
Comment (axecopfire) [inline Portfolio/issues/index.html:111]: Seems strange to wrap a button with an `<a>`
https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

**Comment by axecopfire** [comment]
UI Feedback
- When you hover on the projects, u add a border. However it changes the width and height of the element, so you see it move a bit. Have them remain still so they don't move on hover.
- On any state change like `:hover` any styling changes you do should take some time to transition. https://web.dev/learn/css/transitions/

File: Portfolio/issues/eventListeners.js (hunk 1 above)
Comment (bealecs) [reply Portfolio/issues/eventListeners.js:11]: Noted, thanks

File: Portfolio/issues/index.html (hunk 2 above)
Comment (bealecs) [reply Portfolio/issues/index.html:31]: Noted, fixed

File: Portfolio/issues/index.html (hunk 3 above)
Comment (bealecs) [reply Portfolio/issues/index.html:111]: fixed, never really thought about how that would be weird, thank you.

**Pull request merged**